ArduCopter/system.cpp

```diff
@@ -645,21 +645,21 @@ void Copter::allocate_motors(void)
     }
     AP_Param::load_object_from_eeprom(attitude_control, ac_var_info);
         
-    pos_control = new AC_PosControl(*ahrs_view, inertial_nav, *motors, *attitude_control,
+    pos_control = new AC_PosControl(*ahrs_view, ahrs, *motors, *attitude_control,
```

Comment:
If we are going to change the constructor for AC_PosControl, let's make the argument order consistent with AC_WPNav (i.e. ahrs before ahrs_view).